analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/RuleChecks/NullPointerDereferenceBase.cs

```diff
@@ -31,16 +31,17 @@ public abstract class NullPointerDereferenceBase : SymbolicRuleCheck
 
         protected override ProgramState PreProcessSimple(SymbolicContext context)
         {
-            switch (context.Operation.Instance.Kind)
+            var operation = context.Operation.Instance switch
```

Comment:
To describe what kind of operation is it. Because we have the `context.Operation` itself, but that is not the one we're interested in.
```suggestion
            var instance = context.Operation.Instance switch
```
or
```suggestion
            var reference = context.Operation.Instance switch
```